blocks/network-item/network-item.css

```diff
@@ -0,0 +1,87 @@
+.network-item div.tags {
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: 24px;
+}
+
+.network-item .tags div {
+    display: flex;
+    font-size: var(--body-font-size-xs);
+    margin-right: 1rem;
+    align-items: center;
+}
+
+.network-item .tags div::before {
+    display: inline-block;
+    content: '';
+    background: inherit;
+    background-color: var(--text-color);
+    height: 12px;
+    width: 12px;
+    margin-right: 4px;
+    border-radius: 50%;
+}
+
+.network-item h3.title {
+    font-weight: var(--font-weight-bold);
+}
+
+.network-item .tags div.blue {
+    color: #009CBD; /* oral care */
```

Comment:
Nit: It's good to use existing style in styles.css. If it's not there and it's a reusable one, you can add it.

Reply:
I added the colors to `styles.css`, the names reference the tags name for now, if we can reuse them somewhere else as well we can rename them to something more appropriate